src/cloudWatchLogs/explorer/cloudWatchLogsNode.ts

```diff
@@ -60,6 +62,7 @@ export class CloudWatchLogsNode extends CloudWatchLogsBase {
 
     public constructor(regionCode: string, client = new DefaultCloudWatchLogsClient(regionCode)) {
         super('CloudWatch Logs', regionCode, client)
+        this.contextValue = CONTEXT_VALUE_CLOUDWATCH_LOG_PARENT
```

Comment:
No need for a named value unless you intend to share it and/or it's not obvious what the value means from context:
```suggestion
        this.contextValue = 'awsCloudWatchLogParentNode'
```